src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.calendar__day {
+  box-sizing: border-box;
+  min-width: $width-cell;
+  min-height: $height-cell;
+  background-color: $background-cell;
```

Comment:
The variable `$background-cell` is used but not defined in this file. Ensure it is defined or imported from another SCSS file.